zipkin-server/src/main/java/zipkin2/server/internal/ZipkinHttpCollector.java

```diff
@@ -235,13 +235,19 @@ static HttpData convertRequest(ServiceRequestContext ctx, AggregatedHttpMessage
 
 final class BodyIsExceptionMessage implements ExceptionHandlerFunction {
 
+  static final Logger LOGGER = LogManager.getLogger();
+
   @Override
   public HttpResponse handleException(RequestContext ctx, HttpRequest req, Throwable cause) {
     ZipkinHttpCollector.metrics.incrementMessagesDropped();
+
+    LOGGER.warn("Unexpected error handling request.", cause);
```

Comment:
I'm not sure we want to add this logging in all cases. It seems too verbose to me for bad requests.

Reply:
Good point, moved the logging so it's for cases that aren't bad requests, which we probably do need to see for debugging issues.